chassis/core/src/main/java/com/griddynamics/jagger/user/TestDescription.java

```diff
@@ -30,7 +31,20 @@
     private String version;
     private String name;
 
-    public List<KernelSideObjectProvider<ScenarioCollector<Object,Object,Object>>> getStandardCollectors() {
+    public TestDescription() {
```

Comment:
The one wouldn't be able to create an instance of this object as soon as we have the constructor with parameters.